Handler/Demo.hs

```diff
@@ -0,0 +1,35 @@
+module Handler.Demo where
+
+import Import
+import Model.Site
+
+import Data.Maybe (fromJust, listToMaybe)
+
+getDemoR :: Handler Html
+getDemoR = do
+    root <- getAppRoot
+    siteId <- runDB $ findDemoSite root
+
+    defaultLayout $ do
+        setTitle "Carnival Demo"
+        $(widgetFile "demo")
+
+findDemoSite :: Text -> DB SiteId
+findDemoSite root = do
+    admin <- getAdmin
+    msite <- listToMaybe <$> findSites admin [SiteBaseUrl ==. root]
+
+    entityKey <$> maybe (createDemoSite admin root) return msite
+
+createDemoSite :: UserId -> Text -> DB (Entity Site)
+createDemoSite userId root = createSite userId Site
+    { siteName = "carnival-demo"
+    , siteBaseUrl = root
+    , siteRssAuthor = "Carnival Demo"
+    , siteRssTitle = "Comments on Carnival Demo"
+    , siteRssDescription = "Comments on Carnival Demo"
+    , siteRssLanguage = "en-us"
+    }
+
+getAdmin :: DB UserId
+getAdmin = entityKey . fromJust <$> getBy (UniqueUser "github" "50812")
```

Comment:
This is me and I exist everywhere... except other developer's machines. I had forgotten about that part.

It is strange, but I can't think of how else to do this. Should I create a Nobody user to own the Site? Also (in addition to that question), would it be better to do this in a migration?